app/components/ui/multi-shortcut-editor.tsx

```diff
@@ -60,14 +60,16 @@ export default function MultiShortcutEditor({
     )
   }
 
-  const getErrorMessage = (error: ShortcutError) => {
+  const getErrorMessage = (error: ShortcutError, details?: string) => {
```

Comment:
Nit - but we should probably rename the details to param to something else, maybe `message`, since this is how we're using it.